code/modules/hydroponics/beekeeping/beebox.dm

```diff
@@ -47,9 +44,13 @@
 
 /obj/structure/beebox/Destroy()
 	STOP_PROCESSING(SSobj, src)
-	bees.Cut()
-	honeycombs.Cut()
 	queen_bee = null
+	for(var/mob/living/basic/bee/worker in bees)
+		bees -= worker
+		worker.beehome = null
+		worker.forceMove(get_turf(src))
```

Comment:
`Destroy` should not have side effects, any remaining workers should be qdeleted (which is what is currently being done, any bees in the contents of the box are qdel'd)

You can however also implement `/deconstruct` and put this code here, so if someone bashes the beebox to bits, the bees will flow out instead.